src/plugins/cal-events/cal-events.js

```diff
@@ -80,22 +80,36 @@ var componentName = "wb-calevt",
 	},
 
 	processEvents = function( $elm ) {
-		var settings = $.extend( {}, window[ componentName ], $elm.data( dataAttr ) ),
-			year, month, events, minDate, containerId, $container;
+		var settings = $.extend( {}, window[ componentName ], $elm.data( dataAttr ) ), // Ability to extend the calendar
+			year, month, events, minDate, maxDate, containerId, $container,
+			currDate = new Date(),
+			currYear = currDate.getFullYear(),
+			currMonth = currDate.getMonth();
 
 		events = getEvents( $elm );
 		containerId = $elm.data( "calevtSrc" );
 		$container = $( "#" + containerId ).addClass( componentName + "-cal" );
 
 		minDate = events.minDate;
-		year = settings.year || minDate.getFullYear();
-		month = settings.month || minDate.getMonth();
+		maxDate = events.maxDate;
+
+		if ( minDate.getTime() > currDate.getTime() ) {
```

Comment:
It looks like this already being done in the loop below https://github.com/jilusan/wet-boew/blob/0dd8f5cf4f893a27992ab9138d849bcfa0f1bc8b/src/plugins/cal-events/cal-events.js#L237